packages/hoppscotch-common/src/components/collections/Collection.vue

```diff
@@ -117,6 +117,19 @@
                     }
                   "
                 />
+                <SmartItem
+                  ref="exportPostmanCollectionAction"
+                  :icon="IconDownload"
+                  label="export as postman"
+                  :shortcut="['P']"
+                  :loading="exportLoading"
+                  @click="
+                    () => {
+                      emit('export-postman-collection'),
+                        collectionsType === 'my-collections' ? hide() : null
+                    }
+                  "
+                />
```

Comment:
This isn't required. Export functionality should be unified.

Reply:
We are working on this on a phase by phase basis. This is expected and the PR hasn't left the draft status.